bin/build/test/release/version_info_test.clj

```diff
@@ -17,34 +17,43 @@
    :patch      patch
    :highlights highlights})
 
+;; TODO -- can result in a flakey test if the date changes during this test run
 (def ^:private today (.format (DateTimeFormatter/ofPattern "yyyy-MM-dd") (LocalDate/now)))
 
-(def ^:private test-versions [["39.0" today false ["Super New Fix 1" "Super New Fix 2"]]
-                              ["38.3" "2021-04-01" true ["Fix 1" "Fix 2"]]
-                              ["38.2" "2021-03-21" true ["Older Fix 1" "Older Fix 2"]]
-                              ["38.1" "2021-03-14" true ["Much Older Fix 1" "Much Older Fix 2"]]
-                              ["38.0" "2021-02-15" true ["Super Old Fix 1" "Super Old Fix 2"]]])
+(def ^:private test-versions
+  [["39.0" today false ["Super New Fix 1" "Super New Fix 2"]]
+   ["38.3" "2021-04-01" true ["Fix 1" "Fix 2"]]
+   ["38.2" "2021-03-21" true ["Older Fix 1" "Older Fix 2"]]
+   ["38.1" "2021-03-14" true ["Much Older Fix 1" "Much Older Fix 2"]]
+   ["38.0" "2021-02-15" true ["Super Old Fix 1" "Super Old Fix 2"]]])
 
 (defn- make-version-info [edition versions]
   (let [enterprise? (= edition :ee)]
     {:latest (apply make-version-map (conj (first versions) enterprise?))
-     :older  (map (fn [v]
-                    (apply make-version-map (conj v enterprise?)))
-                  (rest versions))}))
+     :older  (for [v (rest versions)]
+               (apply make-version-map (conj v enterprise?)))}))
+
+(defn- duplicate-version [versions]
+  (into [(first versions)] versions))
+
+(def ^:private previous-version-cases
+  {"with previous versions" (rest test-versions)
+   "with duplicate previous versions" (duplicate-version (rest test-versions))
+   "including the current version" test-versions})
```

Comment:
```suggestion
  {"with previous versions"           (rest test-versions)
   "with duplicate previous versions" (duplicate-version (rest test-versions))
   "including the current version"    test-versions})
```

nit: we try to align the map so that the values are aligned

If you're on emacs I think there are a plugin that do this automatically.